File: C++/Zliczanie linii.cpp
#include <iostream>

int main() 
{
    char character;
    int lineCount = 1;
    while ((character = std::cin.get()) != EOF) 
    {
        if (character == '\n')
            ++lineCount;
    }
    std:: cout << lineCount << '\n';
    return 0;
}

File: Java/Zliczanie linii.java
import java.io.IOException;
import java.io.InputStream;

public class Main 
{
    public static void main(String[] args) 
    {
        InputStream inputStream = System.in;
        int lineCount = 1;
        int character;
        try 
        {
            while ((character = inputStream.read()) != -1) 
            {
                if (character == '\n')
                    lineCount++;
            }
        } catch (IOException e) {
            System.exit(1);
        }
        System.out.println(lineCount);
    }
}
